### PR TITLE
Switches are broken in firefox, need better solution

### PR DESCRIPTION
**Pull request by synbot**
## Description
Switches component needs to be refactored

https://up.kibakoapp.com/IaJhPGI5q6

### CODE REVIEW

File: application/ui/scss/components/forms/_switches.scss [hunk 1]
```diff
@@ -1,111 +1,177 @@
...
+
+  border: none;
+  margin-bottom: $switch-bottom-margin;
+  outline: 0;
+  padding: 0;
+  position: relative;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
```
Comment (canfie1d) [inline application/ui/scss/components/forms/_switches.scss]: order

File: application/ui/scss/components/forms/_switches.scss [hunk 2]
```diff
@@ -1,111 +1,177 @@
...
+
+  label {
+    background: $switch-bg;
+    color: transparent;
+    cursor: pointer;
+    display: block;
+    margin-bottom: ($switch-height-med / 2);
+    position: relative;
+    text-indent: 100%;
+    width: $switch-height-med * 2; height: $switch-height-med;
```
Comment (canfie1d) [inline application/ui/scss/components/forms/_switches.scss]: order

**Comment by canfie1d** [comment]
:interrobang: Comments

**Comment by kylecesmat** [comment]
:star2:

**Comment by canfie1d** [comment]
:pray:

**Comment by pcavagna** [comment]
:hand:

**Comment by pcavagna** [comment]
:+1:

**Pull request merged**